archive_viewer/table_models/curve_model.py

```diff
@@ -223,6 +223,27 @@ def replaceToArchivePlot(self, curve: BasePlotCurveItem, index: QModelIndex, add
         self.plot.removeItem(curve)
         self.endRemoveRows()
 
+    def recursionCheck(self, target: str, rowHeaders: dict) -> bool:
```

Comment:
Style: I would shorten the note in the docstring. Also could you add the parameters and return to the docstring? thanks